src/Workspaces/Core/Desktop/Workspace/Host/TemporaryStorage/TemporaryStorageServiceFactory.MemoryMappedInfo.cs

```diff
@@ -114,7 +114,9 @@ public Stream CreateReadableStream()
                     if (streamAccessor == null)
                     {
                         var rawAccessor = RunWithCompactingGCFallback(info => info._memoryMappedFile.Target.CreateViewAccessor(info.Offset, info.Size, MemoryMappedFileAccess.Read), this);
+#pragma warning disable CA2000 // Dispose objects before losing scope - wrapped within a WeakReference below.
```

Comment:
❗️ Incorrect comment. This is a transfer of ownership to `SharedReadableStream` (and separately, should have been detected as such).